src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -0,0 +1,224 @@
+import { FC, useState } from "react";
+import { LxdInstance } from "types/instance";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+  Select,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { createInstance } from "api/instances";
+import { isClusteredServer } from "util/settings";
+import { useSettings } from "context/useSettings";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchProjects } from "api/projects";
+import { fetchStoragePools } from "api/storage-pools";
+import { fetchClusterMembers } from "api/cluster";
+import { Link } from "react-router-dom";
+import { instanceNameValidation } from "util/instances";
+
+interface Props {
+  instance: LxdInstance;
+  close: () => void;
+}
+
+export interface LxdInstanceDuplicate {
+  instanceName: string;
+  targetProject: string;
+  targetClusterMember?: string;
+  targetStoragePool?: string;
+  allowInconsistent?: boolean;
+  instanceOnly?: boolean;
+  stateful?: boolean;
+}
+
+const DuplicateInstanceForm: FC<Props> = ({ instance, close }) => {
+  const toastNotify = useToastNotification();
+  const { data: settings } = useSettings();
+  const isClustered = isClusteredServer(settings);
+  const controllerState = useState<AbortController | null>(null);
+
+  const { data: projects = [], isLoading: projectsLoading } = useQuery({
+    queryKey: [queryKeys.projects],
+    queryFn: fetchProjects,
+  });
+
+  const { data: storagePools = [], isLoading: storagePoolsLoading } = useQuery({
+    queryKey: [queryKeys.storage],
+    queryFn: () => fetchStoragePools(instance.project),
+  });
+
+  const { data: clusterMembers = [], isLoading: clusterMembersLoading } =
+    useQuery({
+      queryKey: [queryKeys.cluster],
+      queryFn: fetchClusterMembers,
+    });
+
+  const notifySuccess = (instanceName: string) => {
+    const newInstance = (
+      <Link to={`/ui/project/${instance.project}/instance/${instanceName}`}>
+        {instanceName}
+      </Link>
+    );
+    toastNotify.success(
+      <>
+        Created and started cloned instance <b>{newInstance}</b>.
```

Comment:
```suggestion
        Created the instance <b>{newInstance}</b>.
```
We don't start it any more, and maybe this copy is more precise / in alignment with the button / modal text?

I found it a bit confusing to have "duplicate" on the button, and then "cloned" in the notification. Probably we can keep it as the default naming has the `-duplicate` suffix.